guide/src/class/protocols.md

```diff
@@ -6,7 +6,7 @@ Because of the double-underscores surrounding their name, these are also known a
 
 PyO3 makes it possible for every magic method to be implemented in `#[pymethods]` just as they would be done in a regular Python class, with a few notable differences:
 
-- `__new__` and `__init__` are replaced by the [`#[new]` attribute](../class.md#constructor).
+- `__new__` is replaced by the [`#[new]` attribute](../class.md#constructor).
```

Comment:
Not a thing for here but I hope one day maybe we can have `__new__` work exactly like Python (maybe it already does), the `#[new]` attribute might just make sense for some special cases or as a shorthand for `#[pyo3(name = "__new__")]`.

Reply:
Agreed, maybe we can even deprecate and remove it entirely at some point, so that all dunder methods work similarly and the constructor is less special.